tests/security/selinux/sestatus.pm

```diff
@@ -34,6 +34,9 @@ sub run {
     # SELinux by default
     validate_script_output("sestatus", sub { m/SELinux status: .*disabled/ });
 
+    # workaround for "selinux-auto-relabel" in case: auto relabel then trigger reboot
+    assert_script_run("sed -ie \'s/GRUB_TIMEOUT.*/GRUB_TIMEOUT=8/\' /etc/default/grub");
```

Comment:
Why need '-e'?

Reply:
> Why need '-e'?
The "-e" is for safe purpose and I also noticed lots of shell scripts use "-e" even though it might be not needed

Reply:
-e means we can sed >1 items in one command